lisa/features/disks.py

```diff
@@ -56,6 +56,7 @@ def add_data_disk(
         count: int,
         disk_type: schema.DiskType = schema.DiskType.StandardHDDLRS,
         size_in_gb: int = 20,
+        user_lun: int = -1,
```

Comment:
all values are set by caller/user, so call it `lun`.

Reply:
renamed user_lun to lun